src/test/fulcro/client/data_fetch_spec.cljc

```diff
@@ -436,6 +453,26 @@
            "Updates the global loading marker"
            @globally-marked => true)))))
 
+#?(:cljs
+   (specification "Query Response with :initialize (load ... {:initialize true})" :focused
```

Comment:
I think we should also exercise the case where the response doesn't come from the server here, and instead, we see the child initial-state coming from `initial-state`.

Reply:
Agreed. i didn't want to write a lot of tests until we had decided on the API fully. Just enough to show it working.